package.json

```diff
@@ -3,10 +3,11 @@
     "packages/*"
   ],
   "scripts": {
-    "deploy to heroku": "git push heroku master",
+    "deploy branch to heroku": "git push heroku 5-SSE_being_blocked_by_Heroku:master",
```

Comment:
do we need this in master branch?

Reply:
I find it useful to have a specimen on how to deploy not master branch. Ideally branch name should be resolvable via smth like ${current_branch_name}. Do you know if this is possible with git?

Reply:
Maybe try this https://stackoverflow.com/questions/14031970/git-push-current-branch-shortcut ?

Reply:
I don't really know where are those scripts executed, though 🙃

Reply:
Has changed to use 'HEAD' as alias for current branch name